solutions/aspnet/Makefile

```diff
@@ -17,7 +17,7 @@ run: appdir private.pem
 	test -f server.pid && kill -9 `cat server.pid` || true
 	$(MYST) package appdir private.pem config.json
 	myst/bin/$(APPNAME) $(OPTS) & echo $$! > server.pid
-	sleep 15
+	sleep 60
```

Comment:
60 seconds is probably too much for waiting. Why it takes much longer now to boot Mystikos?